setup.py

```diff
@@ -83,6 +83,10 @@
         "bumpversion>=0.5.3,<1",
         "wheel",
         "setuptools>=36.2.0",
+        #Fixing this dependency due to: pytest 3.6.4 has requirement pluggy<0.8,>=0.5, but you'll have pluggy 0.8.0 which is incompatible.
+        "pluggy==0.7.1",
+        #Fixing this dependency due to: requests 2.20.1 has requirement idna<2.8,>=2.5, but you'll have idna 2.8 which is incompatible.
+        "idna==2.7",
```

Comment:
Just as an FYI, I think it would be *ideal* to do `>=2.5,<2.8` here, but since it's only a dev dependency, I'm okay pinning it.